engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/bv/BaseMinMaxValidatorForNumberTest.java

```diff
@@ -74,6 +74,44 @@ protected void testValidatorBigInteger(ConstraintValidator<?, BigInteger> constr
 		assertEquals( constraint.isValid( BigInteger.valueOf( -1560000000 ), null ), isMax );
 		assertEquals( constraint.isValid( BigInteger.valueOf( 1560000000 ), null ), !isMax );
 	}
+	protected void testValidatorByte(ConstraintValidator<?, Byte> constraint, boolean inclusive, boolean isMax) {
+		if ( inclusive ) {
+			assertTrue( constraint.isValid( (byte) 15, null ) );
+		}
+		else {
+			assertFalse( constraint.isValid( (byte) 15, null ) );
+		}
```

Comment:
Wondering if it would be worth it to add a few more tests with the value not being the limit (one lower, one higher)?

Reply:
I've added a test for 14/16 as 15 is a value set in the constraint.